packages/qnamaker/src/commands/qnamaker/kb/export.ts

```diff
@@ -4,7 +4,7 @@ const exportKbJSON = require('./../../../../utils/payloads/exportkb')
 import {Inputs, processInputs} from '../../../utils/qnamakerbase'
 
 export default class QnamakerKbExport extends Command {
-  static description = 'Export a knowledgebase to .json file'
+  static description = 'Echos a knowledgebase .json to stdout'
```

Comment:
```suggestion
  static description = 'Echos a knowledgebase json to stdout'
```

[nit] `.json` is a file extenion. `json` is a data format.